tests/layer_tests/onnx_tests/test_layernorm.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import numpy as np
+import pytest
+from common.layer_test_class import check_ir_version
+from common.onnx_layer_test_class import OnnxRuntimeLayerTest
+
+from unit_tests.utils.graph import build_graph
+
+
+class TestLN(OnnxRuntimeLayerTest):
+    def create_net(self, shape, epsilon, axis, stash_type, ir_version):
+        """
+            ONNX net                   IR net
+            Input->LN->Output   =>    Input->Norm->Power
+        """
+
+        #
+        #   Create ONNX model
+        #
+
+        import onnx
+        from onnx import helper
+        from onnx import TensorProto
+
+        input = helper.make_tensor_value_info('input', TensorProto.FLOAT, shape)
+        output = helper.make_tensor_value_info('output', TensorProto.FLOAT, shape)
+
+        other_shape = shape.copy()[axis:]
+
+        min_val = -127
+        const1 = np.random.randint(min_val, 127, other_shape).astype(float)
+        const2 = np.random.randint(min_val, 127, other_shape).astype(float)
+        print('scale={}'.format(const1))
+        print('bias={}'.format(const2))
+        node_scale_def = helper.make_node(
+            'Constant',
+            inputs=[],
+            outputs=['scale'],
+            value=helper.make_tensor(
+                name='const_tensor',
+                data_type=TensorProto.FLOAT,
+                dims=const1.shape,
+                vals=const1.flatten(),
+            ),
+        )
+
+        node_bias_def = helper.make_node(
+            'Constant',
+            inputs=[],
+            outputs=['bias'],
+            value=helper.make_tensor(
+                name='const_tensor',
+                data_type=TensorProto.FLOAT,
+                dims=const2.shape,
+                vals=const2.flatten(),
+            ),
+        )
+
+        args = dict()
+        args['epsilon'] = epsilon
+        args['axis'] = axis
+        if stash_type != None :
+            args['stash_type'] = stash_type
+
+        node_def = onnx.helper.make_node(
+            'LayerNormalization',
+            inputs=['input', 'scale', 'bias'],
+            outputs=['output'],
+            **args
+        )
+
+        # Create the graph (GraphProto)
+        graph_def = helper.make_graph(
+            [node_scale_def, node_bias_def, node_def],
+            'test_model',
+            [input],
+            [output],
+        )
+
+        # Create the model (ModelProto)
+        onnx_net = helper.make_model(graph_def, producer_name='test_model')
+
+        #   Create reference IR net
+        ref_net = None
+
+        return onnx_net, ref_net
+
+
+    test_data = [
+        dict(shape=[2, 3, 4],    epsilon=9.9999e-05, axis=2, stash_type=None),
```

Comment:
please add at least one test into the pre-commit to avoid breaking.

Reply:
line97-100 is the test case

Reply:
> please add at least one test into the pre-commit to avoid breaking.

line97-100 is the test case

Reply:
No, it is not in the pre-commit. Please add below:

```
    @pytest.mark.precommit
```